### ISSUE TITLE
Audio shifted by 0.1s

### ISSUE THREAD

**Issue by bkbkb** [Aegisub/Aegisub#82]
After loading some videos into Aegisub, the audio is shifted by ~0.1s. I've tested with the "Windows 32 bit portable" versions from http://plorkyeran.com/aegisub/, and it seems it started happening after r8725. Below are screenshots after loading the same video into r8700 and r8725. Only the audio is shifted, and the video stays the same.

r8700:
![image](https://user-images.githubusercontent.com/41873894/43370545-b0c2cb68-934e-11e8-8688-d90d847a052f.png)

r8725:
![image](https://user-images.githubusercontent.com/41873894/43370540-a6690880-934e-11e8-8a07-d6a06ad3949f.png)

Not all videos have this issue. The video I used in the screenshots can be downloaded from [this torrent](https://torrents.ohys.net/download/new/%5BOhys-Raws%5D%20Steins%3BGate%200%20-%2012%20%28MX%201280x720%20x264%20AAC%29.mp4.torrent) (it's episode 12 of Steins;Gate 0, downloaded from ohcy-raws). 

I'm wondering if any of the library updates between r8700 and r8725 might have introduced this. Not sure how the revision numbers are generated, but from the changelog it seems to be between 79fb5cb3 and 5afc543a? Quick link: https://github.com/Aegisub/Aegisub/commits/master?before=524c6114a82157b143567240884de3a6d030b091+210

Does this ring a bell with anyone? If not, I can try digging a bit more after I find time to set up the build.

**Comment by bkbkb** [comment]
I built from source, and looks like the issue is present in the current master. Haven't got time to investigate further though.

![image](https://user-images.githubusercontent.com/41873894/43376977-d7d2c022-938b-11e8-9f16-d0ed1e5acc8c.png)

**Comment by bkbkb** [comment]
Yes, it was an mp4 file, and I didn't touch the file between loading it into the different versions.

**Comment by TurtleWilly** [comment]
> Where are you loading audio from? An mp4 file? I know that mkv has a track delay option that may cause unexpected results, but I don't know about mp4.

Is it supported at all? I shifted a MKV file via  `mkvmerge -o output.mkv -y 1:+1000 input.mkv` and comparing input and output files in AegiSub gives me the audio in the exact same spot as if the shift didn't happened at all. The audio shift works fine in mpv, mediainfo, or VLC, so I know it is in place. I'm using the official release version of AegiSub, which is well… from 2014? Just trying to confirm the current situation here.
